index.html

```diff
@@ -128,6 +128,31 @@ <h4 class="p-heading-icon__title">Vanilla Design</h4>
   </div>
 </div>
 
+<div class="p-strip is-deep is-bordered">
+  <div class="row">
+    <h2>Latest news from our blog</h2>
+    <div class="row">
+      <div class="col-3">
+        <h4><a href="https://ubuntu.com/blog/vanilla-framework-2-0-upgrade-guide">Vanilla Framework 2.0 upgrade guide</a></h4>
+        <p class="p-heading--six">20 June 2019</p>
+      </div>
+      <div class="col-3">
+        <h4><a href="https://ubuntu.com/blog/new-release-vanilla-framework-2-0">New release: Vanilla framework 2.0</a></h4>
+        <p class="p-heading--six">13 June 2019</p>
+      </div>
+      <div class="col-3">
+        <h4><a href="https://ubuntu.com/blog/a-fresh-look-for-releases-ubuntu-com">A fresh look for releases.ubuntu.com</a></h4>
+        <p class="p-heading--six">13 February 2019</p>
+      </div>
+      <div class="col-3">
+        <h4><a href="https://ubuntu.com/blog/vertical-rhythm-and-spacing-improvements-in-vanilla-framework-2-0">Vertical rhythm and spacing in Vanilla Framework 2.0</a></h4>
+        <p class="p-heading--six">15 March 2019</p>
+      </div>
+      <div class="u-fixed-width" style="padding-left: 0px;"><a href="https://ubuntu.com/blog/topics/design" class="p-button--neutral">View more from our blog</a></div>
```

Comment:
```suggestion
      <div class="u-fixed-width u-no-padding--right u-no-padding--left"><a href="https://ubuntu.com/blog/topics/design" class="p-button--neutral">View more from our blog</a></div>
```
Add `u-no-padding--left` instead of the inline style. On mobile the button doesn't look symmetrical, so use `u-no-padding--right` as well.

Reply:
@solazio nice catch will update 👍